src/lib/db/access-store.ts

```diff
@@ -700,18 +772,16 @@ export class AccessStore implements IAccessStore {
 
     async addPermissionsToRole(
         role_id: number,
-        permissions: string[],
+        permissions: Omit<PermissionRef, 'environment'>[],
```

Comment:
Omit the environment just to avoid mistakes, because all permissions should be part of the environment passed as a second parameter.